src/patient-registration/patient-registration.component.tsx

```diff
@@ -115,6 +126,21 @@ export const PatientRegistration: React.FC = () => {
       },
     };
 
+    if (values.contactPersonGivenName || values.contactPersonMiddleName || values.contactPersonFamilyName) {
```

Comment:
If a contact person is being recorded, then we need their given name and family name to be recorded. We should update the acceptance criteria to reflect that and we could use Formik/Yup validation to check that those fields are not empty.